### DISCUSSION TITLE
Apply to be a member of unified

### DISCUSSION THREAD

**Discussion by JounQin** [unifiedjs/unified#193] [General] [answered]
I just found that I'm actually not a member of unified at all, but I hope to be, is there anything I need to apply?

**Comment by ChristianMurphy** [comment, marked as answer]
Members can have several roles:
* contributor
* maintainer
* releaser
* lead

The process for each is explained in https://github.com/unifiedjs/collective/blob/HEAD/members.md

**Reply by wooorm** [reply]
what roles would you want, and on which teams?

**Reply by JounQin** [reply]
According to my [previous work](https://github.com/search?q=author%3AJounQin+org%3Aunifiedjs+org%3Avfile+org%3Asyntax-tree+org%3Aremarkjs+org%3Arehypejs+org%3Aretextjs+org%3Aredotjs+org%3Amicromark+org%3Amdx-js&type=Issues), I'd like to apply to be maintainer of micromark, rehypejs, remarkjs, syntax-tree, vfile and of course unifiedjs, although I don't know whether it is suitable to me.

**Reply by wooorm** [reply]
See: https://github.com/unifiedjs/collective/issues/57.
